geomstats/geometry/matrices.py

```diff
@@ -132,6 +132,23 @@ def random_uniform(self, n_samples=1):
         point = gs.random.rand(n_samples, self.m, self.n)
         return point
 
+    @classmethod
+    def cong(cls, mat_1, mat_2):
+        """Compute the congruent action of mat_2 on mat_1.
+
+        This is `mat_2 mat_1 mat_2^T`
+
+        Parameters
+        ----------
+        mat_1 : array-like, shape=[n_samples, n, n]
```

Comment:
Missing parameters' descriptions. See conversation on slack for documentation's conventions.